drafts/2019-03-22-async-rest-jaxrs-reactive-extensions.adoc

```diff
@@ -11,12 +11,17 @@ blog_description: "Combining reactive programming with JAX-RS 2.1 reactive exten
 = JAX-RS 2.1 Reactive Extensions with RxJava Backpressure
 John Koehler <https://github.com/jkoehler22>
 
-Combining reactive programming with JAX-RS 2.1 reactive extensions make for a powerful asynchronous processing that avoids overwhelming the consumer.   Let's see how.  
+Combining reactive programming with JAX-RS 2.1 reactive extensions make for a powerful asynchronous processing that avoids overwhelming the consumer.   Before we jump into a sample, let's define some of the basic concepts.  
+`Reactive programming` is an asynchronous programming paradigm that deals with data streams and the propagation of change. 
+`RxJava` is a `Reactive Extensions` JVM implementation that enables the writing of event-based, and asynchronous programs.  
+`Backpressure` is when in a processing pipeline, some asynchronous consumers have a build-up and can't process the items fast enough.  
```

Comment:
Backpressure may need a little more explanation.  IIUC, backpressure is what the client does when  the incoming data is more than it can handle.  It might be good to mention that there are different strategies for client (consumers) to implement the backpressure - like buffering, throttling, etc.